Jenkinsfile

```diff
@@ -0,0 +1,22 @@
+
+pipeline {
+    agent label:"docker", docker: "maven:3.3.9-jdk-8"
+
+    environment {
+      MAVEN_OPTS = "-Xmx1024m"
+    }
+
+    stages {
+        stage("Build") {
+            sh 'mvn package -B -Dmaven.test.failure.ignore=true'
+        }
+    }
+
+    postBuild {
+        success {
+            archive "**/target/**/*.jar,**/target/**/*.hpi"
+            junit '**/target/surefire-reports/*.xml'
```

Comment:
Check that, I'm dumb. Forgot that `unstable` won't get set until we do the `junit` anyway! I could see an argument for still doing the junit reports even if the build failed past tests - i.e., findbugs failures or the like - but that's not a big deal.